extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -59,8 +59,10 @@ require([
 			videoLabels = (videoData.labels || '').join(','),
 			videoFeedbackBox,
 			autoplayCookieName = 'featuredVideoAutoplay',
-			autoplay = cookies.get(autoplayCookieName) !== '0' &&
+			autoplayEnabled = cookies.get(autoplayCookieName) !== '0' &&
 				geo.isProperGeo(instantGlobals.wgArticleVideoAutoplayCountries);
+			autoplay = autoplayEnabled && !document.hidden,
+			playerIsAfterInitialPlay = false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "let", "const" or "var" keyword to this declaration of "playerIsAfterInitialPlay" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)